### PR TITLE
Fix Issue 47 (Sub-menu opens nested sub-menu when disabled)

### PR DESCRIPTION
**Pull request by enescang**

### CODE REVIEW

File: packages/mui-nested-menu/src/components/nestedMenuItemsFromObject.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ export function nestedMenuItemsFromObject({
                     parentMenuOpen={isOpen}
                     sx={sx}
                     disabled={disabled}
```
Comment (webzep) [inline packages/mui-nested-menu/src/components/nestedMenuItemsFromObject.tsx:35]: Seeing as we are already passing down `disabled` which contains the same value as `parentMenuDisabled`, perhaps we use `MenuItemProps.disabled` within `NestedMenuItem` to apply the new logic?

**Comment by sumarlidason** [comment]
bump

File: packages/mui-nested-menu/src/components/nestedMenuItemsFromObject.tsx (hunk 1 above)
Comment (webzep) [reply packages/mui-nested-menu/src/components/nestedMenuItemsFromObject.tsx]: Ill remove this change seeing as there is no activity.

**Comment by webzep** [comment]
Changes available on version [3.4.0](https://www.npmjs.com/package/mui-nested-menu/v/3.4.0)

Thanks for contributing!

**Pull request merged**